smf/sled-agent/config-rss.toml

```diff
@@ -38,3 +38,7 @@ addresses = [ "[fd00:1234::7]:12221", "192.168.1.123:12220" ]
 [[request.service]]
 name = "oximeter"
 addresses = [ "[fd00:1234::6]:12223" ]
+
+[[request.service]]
+name = "internal-dns"
+addresses = [ "[fd00:1234::4]:5353" ]
```

Comment:
FYI, with this addition, the RSS spins up an "internal-dns" zone on this Sled.

I confirmed that this service is running and accessible by using:

```bash
cargo run --bin dnsadm -- -a [fd00:1234::4] list-records
```

Reply:
Very cool!